### PR TITLE
Update saving-data.rst

### PR DESCRIPTION
**Pull request by davidyell**
Fixed minor typos

### CODE REVIEW

File: en/orm/saving-data.rst [hunk 1]
```diff
@@ -741,13 +741,13 @@ Applying Application Rules
...
 entities <validating-request-data>`, many applications also have more complex
-validation that should only be applied to after basic validation has completed.
+validation that should only be applied after basic validation has completed.
 These types of rules are often referred to as 'domain rules' or 'application
 rules'. CakePHP exposes this concept through 'RulesCheckers' which are applied
 before entities are persisted. Some example domain rules are:
 
 * Ensuring email uniqueness
-* State transitions or workflow steps, for example updating an invoice's status.
+* State transitions or workflow steps, for example updating an invoices status.
```
Comment (bcrowe) [inline en/orm/saving-data.rst]: I'm pretty sure `invoice's` is correct. The invoice possesses a status, no?

Comment (davidyell) [inline en/orm/saving-data.rst]: Quite possibly.

Comment (markstory) [inline en/orm/saving-data.rst]: Yeah an invoice possesses a status.

**Comment by markstory** [comment]
Possessive fixed in 553ba263469e4c9d6e579479213f518f36ca7a21

**Pull request merged**